contrib/src/main/scala/akka/stream/contrib/SourceGen.scala

```diff
@@ -7,27 +7,35 @@ import akka.stream._
 import akka.stream.scaladsl._
 import akka.stream.stage._
 import com.typesafe.config.ConfigFactory
-import scala.concurrent.duration.Duration
+import scala.concurrent.duration.{ Duration, FiniteDuration }
 
 /**
  * Source factory methods are placed here
  */
 object SourceGen {
+  private[this] lazy val defaultTimeout = Duration.fromNanos(ConfigFactory.load().getDuration("akka.stream.contrib.unfold-flow-timeout").toNanos)
```

Comment:
While I realize it was already done like this in `UnfoldFlow.scala`, we shouldn't use `ConfigFactory.load()` like this. Also it would be better to use the [`getDuration` with 2 parameters](https://lightbend.github.io/config/latest/api/com/typesafe/config/Config.html#getDuration-java.lang.String-java.util.concurrent.TimeUnit-).